packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -268,12 +270,19 @@ class AudioPlayerWithYoutubeSpotify extends Component {
             source={channelToPlay}
             backgroundPhoto={itemPhoto}
             sourceData={this.getAudioSourceInfoToPlay()}
-            customSourceLabel={getChannelLabelToDisplay({
-              channel: channelToPlay,
-              labelValue: audioPlayerChannelLabel,
-              title: `playing ${audioPlayerChannelLabel}`
-            })}
-            linerNotes={null}
+            customSourceLabels={{
+              player: getChannelLabelToDisplay({
+                channel: channelToPlay,
+                labelValue: audioPlayerChannelLabel,
+                title: `playing ${channelToPlay}`
```

Comment:
Just realized this outputs "playing archive", which really should read "playing from source: archive" to be more accurate.

Reply:
you're right. some more wordsmithing can be applied, i will defer to @pezvi